MapboxDirections/MBRouteStep.swift

```diff
@@ -876,6 +882,8 @@ open class RouteStep: NSObject, NSSecureCoding {
         }
         return nil
     }
+    
+    open let voiceInstructions: [VoiceInstruction]?
```

Comment:
Move this property closer to the `instructions` property. Also call it `instructionsSpokenAlongStep` or `instructionsSpokenInAdvance` to clarify that these instructions pertain to the following step, unlike the `instructions` property.